src/service/discovery.js

```diff
@@ -178,14 +178,29 @@ const fetchInitialDiscoveryTopics = async (limit = 50, page = 0) => {
  * @property {FetchInitialDiscoveryUsersResponse.SuggestedUsers[]} [suggestedUsers]
  */
 
+/**
+ * @typedef {Object} FetchInitialDiscoveryUsersOptionalParams
+ * @property {string[]} excudedUserIds
+ * @property {string[]} excludedUserNames
+ */
+
 /**
  * @param {Number} limit
  * @param {Number} page
+ * @param {boolean} isAnon
+ * @param {FetchInitialDiscoveryUsersOptionalParams} optionalParams
  * @returns {FetchInitialDiscoveryUsersResponse}
  */
-const fetchInitialDiscoveryUsers = async (limit = 50, page = 0, isAnon = false) => {
+const fetchInitialDiscoveryUsers = async (
+  limit = 50,
+  page = 0,
+  isAnon = false,
+  optionalParams = {}
+) => {
   const body = {
-    page
+    page,
+    excluded_user_names: optionalParams?.excludedUserNames,
+    excluded_user_ids: optionalParams?.excudedUserIds
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The same typo is present here as well. It should be `excludedUserIds` instead of `excudedUserIds`.
```diff
-    excluded_user_ids: optionalParams?.excudedUserIds
+    excluded_user_ids: optionalParams?.excludedUserIds
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->